tools/fast/bin/localnet/main.go

```diff
@@ -66,8 +71,27 @@ func init() {
 		minerExpiryArg     = minerExpiry.Text(10)
 	)
 
+	// We default to the binary built in the project directory, fallback
+	// to searching path.
+	binpath, err = getFilecoinBinary()
```

Comment:
personally id distinguish between an unexpected error (error) and not finding the file (which you could signal with a bool return value). in the first case this should bail out. in the second, continue into the conditional below.

Reply:
The issue is I don't want to bail out before parsing the flags, as I want to still handle `-help` even if this errors, and if the user passes in a value on the flag, the error is mute.